app/views/templates/_template.html.slim

```diff
@@ -1,12 +1,13 @@
-tr.info
+tr.info ng-repeat="template in templates | filter:query "
   td
-    =link_to template_path(template), rel: 'tooltip', title: 'Show template'
-      = template.name
+    | {{template.name}}
   td
-    .btn-toolbar aria-label="..." role="toolbar"
-      = link_to template_path(template), class: "btn btn-sm btn-info" do
-        i.glyphicon.glyphicon-eye-open Show
-      = link_to edit_template_path(template), class: "btn btn-sm btn-warning" do
-        i.glyphicon.glyphicon-edit Edit
-      = link_to template_path(template),   method: :delete, data: { confirm: 'Are you sure?' }, class: "btn btn-sm btn-danger" do
-        i.glyphicon.glyphicon-remove Delete
+    a ng-href="/templates/{{template.id}}"
+      button class="btn btn-sm btn-info glyphicon glyphicon-eye-open"
+        | Show
+    a ng-href="/templates/{{template.id}}/edit"
```

Comment:
In my opinion it will be better to make link in {{template.name}}, and remove this icon. In this case the table will be in style with questions table.